src/components/Dashboard/Alert/AlertItem.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+import { Col, Row, Icon } from 'patternfly-react';
+
+const getSeverityIcon = severity => {
+  switch (severity) {
+    case 'critical':
+      return <Icon type="fa" name="exclamation-circle" className="kubevirt-alert__icon-critical" />;
+    case 'warning':
+    default:
+      return <Icon type="fa" name="exclamation-triangle" className="kubevirt-alert__icon-warning" />;
+  }
+};
+
+export const AlertItem = ({ alert }) => (
+  <Row className="kubevirt-alert__item">
+    <Col lg={1} md={1} sm={1} xs={1}>
```

Comment:
can't we rather use css with a margin here?

Reply:
done